### DISCUSSION TITLE
PC端能否取消最高显示

### DISCUSSION THREAD

**Discussion by TDraking** [xfangfang/wiliwili#369] [Ideas]
全屏的情况下，任何软件的页面都会被wili覆盖，即使设置里的最高层级显示处于关闭状态。
能否让wili像普通应用一样可以被其他页面覆盖

**Comment by xfangfang** [comment]
@TDraking 我收到过几次类似的反馈，可以在历史的issue里查找到，欢迎提交PR，这个问题可能需要调整GLFW的代码来实现，还可能和系统或连接的显示器有关，从我这边测试的结果看是可以正常显示输入法和win11的屏幕键盘的。
因为我日常用windows不多，而且感觉全屏下并没有造成什么使用上的困扰，所以我就不去研究这个问题了，不过欢迎大家的PR。

**Comment by TDraking** [comment]
不好意思，可能是我没说清楚，我的意思是wili会挡住其他应用窗口，
例如，打开了全屏的wili，此时通过win+d快捷键只会显示出任务栏，桌面被wili遮挡；通过alt+tab切换其他应用时，其他应用也会被wili遮挡。

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: xfangfang ***@***.***>
发送时间: Sunday, June 30, 2024 11:43:27 AM
收件人: xfangfang/wiliwili ***@***.***>
抄送: TDraking ***@***.***>; Mention ***@***.***>
主题: Re: [xfangfang/wiliwili] PC端能否取消最高显示 (Discussion #369)


@TDraking<https://github.com/TDraking> 我收到过几次类似的反馈，可以在历史的issue里查找到，欢迎提交PR，这个问题可能需要调整GLFW的代码来实现，还可能和系统或连接的显示器有关，从我这边测试的结果看是可以正常显示输入法和win11的屏幕键盘的。

―

**Reply by xfangfang** [reply]
如果有改进的方法欢迎提交PR。
